package/scripts/presto_client.py

```diff
@@ -80,7 +80,8 @@ def ensure_nodes_are_up(client):
         raise RuntimeError('Presto server failed to start within \
 {0} seconds.'.format(RETRY_TIMEOUT))
 
-
+# This class was copied more or less verbatim from
```

Comment:
The line right below shows you where I got this code from. I use it in the smoketest function, which is at the top of this file. That function accepts a PrestoClient() instance as an argument and goes on to validate the installation. In turn the smoketest function is called from the Coordinator class (in the presto_coordinator.py file) in the start method.